src/Octoshift/Services/ReclaimService.cs

```diff
@@ -173,9 +175,17 @@ public virtual async Task ReclaimMannequins(string[] lines, string githubTargetO
                 continue;
             }
 
-            var result = await _githubApi.ReclaimMannequin(githubOrgId, userid, claimantId);
-
-            HandleResult(login, claimantLogin, mannequin, claimantId, result);
+            if (skipInvitation)
+            {
+                //TODO: Check if org is emu before continuing, throw error if not
+                dynamic result = await _githubApi.ReclaimMannequinsSkipInvitation(githubOrgId, userid, claimantId);
```

Comment:
dynamic should almost never be used in C#. It's really only appropriate in some very niche cases (I've never had to use it in 20 years of doing c#).

```suggestion
                var result = await _githubApi.ReclaimMannequinsSkipInvitation(githubOrgId, userid, claimantId);
```